components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -345,4 +347,37 @@ public boolean doPostUpdateRoleName(String roleName, String newRoleName, UserSto
         }
         return attributes;
     }
+
+    @Override
+    public boolean doPostGetUserClaimValues(String userName, String[] claims, String profileName, Map<String, String>
+            claimMap, UserStoreManager storeManager) throws UserStoreException {
+
+        if (!isEnable()) {
+            return true;
+        }
+
+        if (log.isDebugEnabled()) {
+            log.debug("doPostGetUserClaimValues getting executed in the SCIMUserOperationListener for user: " +
+                    userName);
+        }
+
+        if (claimMap == null) {
+            claimMap = new HashMap<>();
+        }
+        // Check whether http://wso2.org/claims/identity/isReadOnlyUser claim is requested.
+        boolean userMutabilityRequested = false;
+        for (String claim : claims) {
+            if (claim.contains(SCIMCommonConstants.READ_ONLY_USER_CLAIM)) {
```

Comment:
contains or equals? this will match to http://wso2.org/claims/identity/isReadOnlyUserzzzzz as well.

Reply:
Can simplify to Arrays.asList(claims).contains(SCIMCommonConstants.READ_ONLY_USER_CLAIM)